src/modules/logger/logger.cpp

```diff
@@ -519,6 +510,7 @@ bool Logger::initialize_topics(MissionLogType mission_log_mode)
 void Logger::run()
 {
 	PX4_INFO("logger started (mode=%s)", configured_backend_mode());
+	update_params(true);
```

Comment:
This is not needed, also the `force` can be dropped.

Reply:
done